eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -433,4 +448,17 @@ private function getNumberOfCPUCores()
 
         return $cores;
     }
+
+    /**
+     * @param OutputInterface $output
+     * @param array $unindexableContentIds
+     */
+    private function printUnindexableContentIds(OutputInterface $output, $unindexableContentIds)
+    {
+        $output->writeln('');
+        $output->writeln(sprintf(
+            '<error>Indexing failed on some content items, try running command with "--iteration-count=1 --content-ids=<ids>": %s</error>',
+            implode(', ', $unindexableContentIds)
```

Comment:
If we skip the space after `,` it will be possible to copy paste the output to --content-ids argument.